Content/1-Introduction.tex

```diff
@@ -6,13 +6,14 @@ \chapter{\iflanguage{ngerman}{Einleitung}{Introduction}}
 As an useful aid in all scientific work following book is recommended: \cite{deininger1992studienarbeiten}
 
 \dots
-\todo{Rewrite this seciton}
+\todo{Rewrite this section}
 \nomenclature{IAR-IPR}{Institute for Anthropomatics and Robotics (IAR) - Intelligent Process Control and Robotics (IPR)}
 \Blindtext
 
-\subsection{Introduction Sub}
```

Comment:
This is needed for compatibility with Seminars

Reply:
Is it possible to add an additional `\section{}` entry before?